code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -15,23 +15,8 @@
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
-	if(health > 0)
-		return
-	else
-		feedback_set_details("megafauna_kills","[initial(name)]")
-		..()
-
-/mob/living/simple_animal/hostile/megafauna/gib()
```

Comment:
Yes. Honestly, I just hate flat immunities. If you have a WoD, you deserve your one guaranteed boss kill. 

And it wouldn't work on Legion, current end boss, for the reasons I already stated.